src/lib.rs.in

```diff
@@ -609,6 +619,13 @@ class IronSdkAdvanced {
     /// @param a0 `documentData`   - bytes of the document to encrypt
     /// @param a1 `encryptOpts`    - optional document encrypt parameters
     method IronSdkAdvanced::document_encrypt_unmanaged(&self,_: &[i8], _: &DocumentEncryptOpts) -> Result<DocumentEncryptUnmanagedResult, String>; alias documentEncryptUnmanaged;
+    /// Decrypt the provided encrypted document with the encrypted document encryption keys (EDEKs).
+    /// 
+    /// @param a0 'encrypted_data' - bytes of encrypted document. Should be the same bytes returned from {@link documentEncryptUnmanaged(byte[], DocumentEncryptOpts)}
+    /// @param a1 'encrypted_deks' - encrypted document encryption keys
```

Comment:
If we are going to link to the data, should we link to the edeks too?

Reply:
Probably.

Reply:
Added a link to documentEncryptUnmanaged()